app/models/concerns/indexable.rb

```diff
@@ -30,6 +30,21 @@ def create_index
         body:  { settings: settings, mappings: mappings })
     end
 
+    def update_metadata(version)
+      ES.client.index(
+        index: index_name,
+        type: 'metadata',
+        id: 0,
+        body: { version: version, time: DateTime.now.utc })
+    end
+
+    def stored_metadata
+      ES.client.search(
+        index: index_name,
+        type: 'metadata',
+        body: {query: {match: {_id: 0}}})['hits']['hits'][0]['_source'].symbolize_keys rescue {}
```

Comment:
BTW, suppressing exceptions like this is [considered bad form](https://github.com/bbatsov/ruby-style-guide#dont-hide-exceptions). You need to ask yourself: under what circumstances do I expect this statement to fail, and can I rewrite it to avoid the failure? If not, you should be more explicit about the type of exception you expect to catch.